website/docs/getting-started/cli-commands.md

```diff
@@ -0,0 +1,160 @@
+## GraphQL-Mesh Commands
+
+Graphql-mesh exposes a number of commands that can be used to determine interactions as well as startup behavior.
+
+When running mesh commands please be aware that some will work differently based on the context you are executing them in. Global installed CLI vs local CLI do not return the exact same results in all circumstances.
+
+Global installation:
+```
+yarn global add graphql @graphql-mesh/cli
+```
+
+If you have a global install of the cli and you can use `graphql-mesh` in terminal directly.
+
+As indicated the result of the command may differ based on whether you run the global CLI command or the locally installed CLI inside your target project's root directory.
+The safest method is the local execution.
+
+To run project specific commands CLI:
+
+> windows     `node node_modules\\.bin\\graphql-mesh dump-schema --output ./schema.json`
+
+> mac/linux   `node node_modules/.bin/graphql-mesh dump-schema --output ./schema.json`
+
+
+
+To run a project specific commands repeatedly it is more practical to add a script reference inside your `package.json` like so:
+
+```json
+script: {
+    "dump": "node_modules\\.bin\\graphql-mesh dump-schema --output ./schema.json",
+```
+
+
+## Global Options
```

Comment:
Out of curiosity, are these options global in the sense that they are available for all commands, regardless of whether the command is based on the local or global CLI? Or are they only available when running the globally-installed CLI?

I think it may not be 100% obvious since you mentioned that you may get different results when using the different install options.

Reply:
These are global as in they  are available to all commands.
I will clarify.